tests/high_level_api.rs

```diff
@@ -87,36 +87,36 @@ wasmer_backends! {
         assert_eq!(
             exports,
             vec![
-                export::ExportDescriptor {
+                export::ExportType {
                     name: "test-table",
-                    ty: export::ExternDescriptor::Table(TableDescriptor {
+                    ty: export::ExternType::Table(TableType {
                         element: ElementType::Anyfunc,
                         minimum: 2,
                         maximum: None,
                     }),
                 },
-                export::ExportDescriptor {
+                export::ExportType {
                     name: "test-global",
-                    ty: export::ExternDescriptor::Global(GlobalDescriptor {
```

Comment:
This field `ty` should be named `desc` for descriptor. The two places descriptors show up in the spec are as part of export types and import types (see 2.5.10 and 2.5.11). The export type is {name, desc} and the desc is {func | table | mem | global}.

Reply:
Very good catch!  I just discussed the use of `desc` with Syrus now and we looked at the Wasm C API was doing as a reference and it seems that it just generally avoids indices in favor of other types, like `wasm_exporttype` here (https://github.com/WebAssembly/wasm-c-api/blob/master/include/wasm.h#L295).  So we're going to defer this for now, but this is something worth following up on.